### DISCUSSION TITLE
Does this calendar support some sort of Resource View

### DISCUSSION THREAD

**Discussion by luis-cortina** [nhn/tui.calendar#1433] [Q&A]
I'm looking to implement something like this? Is it possible to do it with this library out of the box?
![telegram-cloud-photo-size-1-5170543508421979264-y](https://github.com/nhn/tui.calendar/assets/25346332/7ede95e9-188f-4928-ab26-179ced48ab26)

**Comment by paulogustavo3301** [comment]
qual a sua sugestão mano?

Em qua., 13 de dez. de 2023 15:50, Luis Cortina ***@***.***>
escreveu:

**Comment by toomanylogins** [comment]
I am looking to do something similar. Was this calendar a suitable solution ?
